### PR TITLE
Get started guide for RIF Relay Sample dApp SDK

### PR DESCRIPTION
**Pull request by zdaodu-iov**
## What

- The guide includes the process to get started with the RIF Relay Sample dApp SDK.

## Why

- The purpose of these changes is to facilitate developers and users in effectively utilizing the RIF Relay
- The guide aims to lower the entry barrier for new users and enhance the understanding of the SDK’s capabilities.

## Refs

- (optional: include links to other issues, PRs, tickets, etc)

### CODE REVIEW

**Review by antomor — COMMENTED** [review]
@zdaodu-iov Thank you for the proposal, but I think that it's not a good idea to include the steps required to run an RSKj node into the relay guide. IMHO This is something that should go in a common place, likely in the developer portal and eventually, include here as a link. Adding this guide here may mean that we need to maintain it in future, and I'd avoid it

**Comment by zdaodu-iov** [comment]
@antomor I agree. Actually this PR is still a work in progress. I have a [PR](https://github.com/rsksmart/devportal/pull/944) open in the dev portal to add these instructions and then I'll link to it from here.

**Comment by antomor** [comment]
> @antomor I agree. Actually this PR is still a work in progress. I have a [PR](https://github.com/rsksmart/devportal/pull/944) open in the dev portal to add these instructions and then I'll link to it from here.

Great, thank you! I'll review it again once it's ready 👍🏼

**Comment by zdaodu-iov** [comment]
@antomor the PR is now ready for review, thanks!

File: Readme.md [hunk 1]
```diff
@@ -23,6 +23,37 @@ Rif Relay is built in modules, the entire system is made up by 4 modules.
...
+
+### Run the Rootstock Node
+The Rootstock node enhances network security, smart contract functionality, privacy, and access to real-time data. It also supports the RSK ecosystem and offers incentives like transaction fees or block rewards. Setup the node on `--regtest mode` using [JAR package](https://dev.rootstock.io/rsk/node/install/operating-systems/java/) or [Docker container](https://dev.rootstock.io/rsk/node/install/operating-systems/).
+
+### Add Network to Metamask
+To interact with the Rootstock network, you need to add it to Metamask. Follow the steps in [How to add Metamask to Rootstock](https://dev.rootstock.io/develop/wallet/use/metamask/) to add the Regtest Network since we're using the node on `--regtest mode`.
+
+### Setup RIF Relay Contracts and Server
+The setup of RIF Relay contracts and server is a multi-step process. It involves deploying on-chain components, meeting installation requirements, and using various development tools. For detailed instructions and necessary resources for this setup, see [Setup RIF Relay Contract and Server guide](https://dev.rootstock.io/guides/rif-relay/deployment/).
```
Comment (antomor) [inline Readme.md:48]:
```suggestion
The setup of RIF Relay Contracts and Server is a multi-step process. It involves deploying on-chain components, meeting installation requirements, and using various development tools. For detailed instructions and necessary resources for this setup, see [Setup RIF Relay Contract and Server guide](https://dev.rootstock.io/guides/rif-relay/deployment/). 
```

File: Readme.md [hunk 2]
```diff
@@ -23,6 +23,37 @@ Rif Relay is built in modules, the entire system is made up by 4 modules.
...
+
+### Setup RIF Relay Contracts and Server
+The setup of RIF Relay contracts and server is a multi-step process. It involves deploying on-chain components, meeting installation requirements, and using various development tools. For detailed instructions and necessary resources for this setup, see [Setup RIF Relay Contract and Server guide](https://dev.rootstock.io/guides/rif-relay/deployment/). 
+For specific resources, see:
+- [RIF Relay Server](https://github.com/rsksmart/rif-relay-server).
+- [RIF Relay Contracts](https://github.com/rsksmart/rif-relay-contracts).
+
+### Configure and Run the RIF Relay Sample dApp
+
+This is a sample dApp to show you how to submit relayed transactions to the Rootstock blockchain using the [RIF Relay SDK](https://github.com/infuy/relaying-services-sdk). Connect to the dApp with MetaMask for signing transactions with the account that owns the Smart Wallets. For more detials, see [RIF Relaying Services SDK sample dApp](https://github.com/rsksmart/rif-relay-sample-dapp).
```
Comment (antomor) [inline Readme.md:55]:
```suggestion
This is a sample dApp to show you how to submit relayed transactions to the Rootstock blockchain using the [RIF Relay Client](https://github.com/rsksmart/rif-relay-client). Connect to the dApp with MetaMask for signing transactions with the account that owns the Smart Wallets. For more detials, see [RIF Relay Sample dApp](https://github.com/rsksmart/rif-relay-sample-dapp).
```

**Review by antomor — CHANGES_REQUESTED** [review]
@zdaodu-iov thank you for the changes, I've left some minor improvements.

**Review by antomor — APPROVED** [review]

File: Readme.md [hunk 3]
```diff
@@ -23,6 +23,10 @@ Rif Relay is built in modules, the entire system is made up by 4 modules.
 
 Each module has instructions for development and usage.
 
+## Getting Started with the RIF Relay Sample dApp SDK
+
+We've created a sample dApp to showcase how users can submit relayed transactions to the RSK blockchain using the RIF Relay Sample dApp SDK. For details, see [RIF Relaying Services SDK sample dApp](https://github.com/rsksmart/rif-relay-sample-dapp).
```
Comment (owans) [inline Readme.md:28]:
```suggestion
We've created a sample dApp to showcase how users can submit relayed transactions to the Rootstock blockchain using the RIF Relay Sample dApp SDK. For details, see [RIF Relaying Services SDK sample dApp](https://github.com/rsksmart/rif-relay-sample-dapp).
```

**Review by owans — APPROVED** [review]

**Review by antomor — COMMENTED** [review]
@zdaodu-iov @owans Not sure if this was intended, but almost all of the previous changes got reverted

**Review by antomor — APPROVED** [review]

**Pull request merged**